frontend/src/components/communication/Post.js

```diff
@@ -115,28 +118,39 @@ export default function Post({
             />
           </Link>
           <span className={classes.messageWithMetaData}>
-            <div className={classes.metadata}>
-              <Link
-                color="inherit"
-                href={getLocalePrefix(locale) + "/profiles/" + post.author_user.url_slug}
-                target="_blank"
-              >
-                <Typography variant="body2" className={classes.username}>
-                  {post.author_user.first_name + " " + post.author_user.last_name}
+            {type != "preview" && (
```

Comment:
```suggestion
            {type !== "preview" && (
```
Always use `===` or `!==` unless you concrete don't want to do that. It's a bit safer since it makes it less likely to have an unexpected equals.

Reply:
Same goes for all other instances of `==` or `!=`